cli/src/commands/describe.rs

```diff
@@ -75,6 +75,9 @@ pub(crate) struct DescribeArgs {
     /// for all of them.
     #[arg(long)]
     stdin: bool,
+    // TODO: Delete in jj 0.40.0+
+    /// (deprecated together with --reset-author, use `jj metaedit`)
+    ///  
```

Comment:
nit: remove trailing whitespace

(I think it's also good to just add `hide = true`.)